File: Py-programming/casual/hello.py
def sum(a, b):
    return a+b

def diff(a, b):
    return a-b

def mult(a, b):
    return a*b

def div(a,b):
    return a/b


def operator():
    num = int(input("isnert:\n1 for sum \n2 for diff \n3 for mult \n4 for div\n>> "))
    
    if num not in [1, 2, 3, 4]:
        return "invalid number"

    a = float(input("insetr a number: "))
    b = float(input("insert a number: "))

    if num == 1:
        return sum(a, b)
    elif num == 2:
        return diff(a, b)
    elif num == 3:
        return mult(a, b)
    elif num == 4:
        return div(a, b)


result = operator()
print(f"the result is: {result}")

File: C-programming/study/0-starting/hello.c
#include <stdio.h>

int main() {

	char i = 10;

	switch (i)
	{
	case 5: case 6: case 7: case 8: case 9: case 10:
		printf("the number is: %d", i);
		break;
	
		case 11:
			printf("the number is: %c", i);
	default:
		break;
	}	
}